src/status_im/contexts/wallet/wallet_connect/effects.cljs

```diff
@@ -0,0 +1,43 @@
+(ns status-im.contexts.wallet.wallet-connect.effects
+  (:require [re-frame.core :as rf]
+            [status-im.contexts.wallet.wallet-connect.utils :as wallet-connect.utils]
+            [taoensso.timbre :as log]))
+
+(rf/reg-fx
+ :effects.wallet-connect/init
+ (fn [{:keys [on-success on-fail]}]
+   (-> (wallet-connect.utils/init)
+       (.then on-success)
```

Comment:
This is clean but as a team we decided to shift to promesa.

Reply:
done